mitiq/zne/scaling/folding.py

```diff
@@ -124,7 +116,7 @@ def _check_foldable(circuit: Circuit) -> None:
         )
 
 
-def squash_moments(circuit: Circuit) -> Circuit:
+def _squash_moments(circuit: Circuit) -> Circuit:
```

Comment:
No change needed, but we really need to make submodule structure consistent. Here, `squash_moments` is never imported in `zne.scaling.__init__.py`, so it is private and the underscore is extraneous. In other places though (`zne/inference.py`) the underscore is needed because the module structure is different.